tests/unit_tests/model/test_mirrors.py

```diff
@@ -3,26 +3,189 @@
 import logging
 
 import pytest
+from astropy.utils.diff import report_diff_values
 
-import simtools.utils.general as gen
-from simtools.model.mirrors import Mirrors
+from simtools.model.mirrors import InvalidMirrorListFile, Mirrors
 
 logger = logging.getLogger()
 logger.setLevel(logging.DEBUG)
 
 
-def test_read_list(db, io_handler):
-    test_file_name = "mirror_CTA-LST-flen_grouped.dat"
-    db.export_file_db(
-        db_name=db.DB_CTA_SIMULATION_MODEL,
-        dest=io_handler.get_output_directory(sub_dir="model", dir_type="test"),
-        file_name=test_file_name,
+def test_read_mirror_list_from_sim_telarray(db, io_handler):
+    mirror_list_file = io_handler.get_input_data_file(
+        file_name="mirror_list_CTA-N-LST1_v2019-03-31_rotated_simtel.dat",
+        test=True,
     )
-    mirror_list_file = gen.find_file(
-        test_file_name, io_handler.get_output_directory(sub_dir="model", dir_type="test")
+    logger.info(f"Using mirror list {mirror_list_file}")
+    mirrors = Mirrors(mirror_list_file)
+    assert 198 == mirrors.number_of_mirrors
+    assert 151.0 == pytest.approx(mirrors.mirror_diameter.value)
+    assert 3 == mirrors.shape_type
+
+
+def test_read_mirror_list_from_ecsv(io_handler, tmp_test_directory, caplog):
```

Comment:
Technically each case below should have it's own test function. It simplifies hunting down issues in the future and running only specific cases when you find them. It can sometimes lead to duplications, but those are usually really minimised when using fixtures. You can look at other test files to see examples. 
Again, this isn't really a blocker for this PR. We can improve it in a separate PR if you want to merge this one first.